samples/stylus-sample/package.json

```diff
@@ -44,10 +44,25 @@
     "lodash": "^4.10.1",
     "milligram-stylus": "^1.3.0",
     "react-router-config": "^1.0.0-beta.4",
-    "react-router-dom": "^4.3.1"
+    "react-router-dom": "^4.3.1",
+    "electrode-archetype-opt-less": "^1.0.2",
+    "electrode-archetype-opt-postcss": "^1.0.4",
+    "electrode-archetype-opt-pwa": "^1.0.6",
```

Comment:
are these three required for run time?

```
    "electrode-archetype-opt-less": "^1.0.2",
    "electrode-archetype-opt-postcss": "^1.0.4",
    "electrode-archetype-opt-pwa": "^1.0.6",
```

Reply:
I could definitely use your help determining the default behavior for `devOnly`. For instance, Less definitely has a run-time component even if it isn't particularly performant (  http://lesscss.org/usage/#using-less-in-the-browser ). Should I take from this comment that you definitely want all 3 to be `devOnly`?

Reply:
Exist behavior is that all opt packages, except opt-react and opt-inferno are dev only.  It's safer to start with dev only and determine what run time is needed.  We may need to create runtime packages, or add them for the app directly.